01-node-tutorial/answers/writeWithPromiseThen.js

```diff
@@ -0,0 +1,21 @@
+const {writeFile,readFile} = require("fs").promises;
+
+writeFile('./temporary/temp.txt', "This is line one")
+.then(() => {
+    return writeFile('./temporary/temp.txt', "This is line two", { flag: 'a' });
```

Comment:
I don't think this is a huge deal either but it says on line 2 but it actually is all on the same line. You either need to user a different method or make sure to add a new line character.